submissions/Oleksii_Drohachov/Memory_pair_game/script.js

```diff
@@ -0,0 +1,220 @@
+const mainContainer = document.querySelector(".container");
+const modal = document.querySelector(".modal__container");
+const modalBtn = document.querySelector(".modal__btn");
+const gameboard = document.querySelector(".gameboard");
+const scoreCount = document.querySelector(".score__container");
+
+renderModal(
+  "You are welcome, my friend ! Wanna play some memory game to endure your mind and become more powerfull ? Try to find pairs of cards as fast as you can! Tap the button to begin... ",
+  "START THE GAME"
+);
+
+function renderModal(titleText, btnText) {
+  const modalWrapper = document.createElement("div");
+  modalWrapper.classList.add("modal__wrapper");
+  const modalTitle = document.createElement("p");
+  modalTitle.classList.add("modal__title");
+  modalTitle.textContent = titleText;
+  const modalBtn = document.createElement("button");
+  modalBtn.classList.add("modal__btn");
+  modalBtn.textContent = btnText;
+  modalBtn.addEventListener("click", startGame);
+  const closeBtn = document.createElement("button");
+  closeBtn.classList.add("modal__btn");
+  closeBtn.textContent = "NO, THANKS";
+  closeBtn.addEventListener("click", hideModal);
+
+  modalWrapper.append(modalTitle);
+  modalWrapper.append(modalBtn);
+  modalWrapper.append(closeBtn);
+
+  modal.append(modalWrapper);
+  modal.classList.add("shown");
+}
+
+function renderModalForReset() {
+  renderModal("Are you sure, you want to restart the game?", "RESET");
+}
+
+function renderModalAfterWin() {
+  renderModal(
+    "Good job my friend! You became even stronger! Want to start one more time?",
+    "RESET"
+  );
+}
+
+function hideModal() {
+  modal.classList.remove("shown");
+  modal.innerHTML = "";
+}
+
+function startGame() {
+  scoreCount.innerHTML = "";
+  gameboard.innerHTML = "";
+  modal.innerHTML = "";
+  modal.classList.remove("shown");
+  prepareGame();
+}
+
+function prepareGame() {
+  const cardsArray = [
+    {
+      name: "beast",
+      path: "img/Beast.jpg",
+    },
+    {
+      name: "beast",
+      path: "img/Beast.jpg",
+    },
+    {
+      name: "death",
+      path: "img/Death.jpg",
+    },
+    {
+      name: "death",
+      path: "img/Death.jpg",
+    },
+    {
+      name: "fire",
+      path: "img/Fire.jpg",
+    },
+    {
+      name: "fire",
+      path: "img/Fire.jpg",
+    },
+    {
+      name: "life",
+      path: "img/Life.jpg",
+    },
+    {
+      name: "life",
+      path: "img/Life.jpg",
+    },
+    {
+      name: "light",
+      path: "img/Light.jpg",
+    },
+    {
+      name: "light",
+      path: "img/Light.jpg",
+    },
+    {
+      name: "metal",
+      path: "img/Metal.jpg",
+    },
+    {
+      name: "metal",
+      path: "img/Metal.jpg",
+    },
+  ];
+  let shuffled = [...cardsArray];
```

Comment:
```suggestion
  let shuffled = [...cardsArray, ...cardsArray];
```
Also, if you don't reassign it use const instead.